deploy/olm-catalog/windows-machine-config-operator/manifests/windows-exporter-role_binding.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
Is there no way to define this inside the CSV? Moreover this is now missing from the `deploy` folder which implies, we have to use OLM to deploy the operator and can't do a direct deployment.